docs/quick_start/AA_hexane_ethanol_example.rst

```diff
@@ -0,0 +1,41 @@
+Example of an All-Atom (AA) Hexane and Ethanol System
+========================
+
+**Note**: `foyer <https://foyer.mosdef.org/en/stable/>`_ is used in conjunction with **mBuild** in the following example to demonstrate how all the `MoSDeF <https://mosdef.org>`_ software integrates to generate a simulation box.
+
+Import the required mbuild package.
+
+.. code:: ipython3
+
+    import mbuild as mb
+
+
+Construct an all-atom (AA) hexane and ethanol using the OPLS-AA force field (FF),
+which is comes as a standard `foyer <https://foyer.mosdef.org/en/stable/>`_ force field (FF).
+The hexane and ethanol molecules will be created using `smiles strings <https://www.daylight.com/dayhtml/doc/theory/theory.smiles.html>`_. 
+The hexane and ethanol residues will be named HEX and ETO, respectively.  
+Lastly, the hexane and ethanol molecule's configuration will be energy minimized, properly reorienting the molecule to the specified FF, which is sometimes needed for some simulation engines to ensure the initial configuration energy is not too high.     
+
+**Note**: The energy minimize step requires the `foyer <https://foyer.mosdef.org/en/stable/>`_ package. 
+
+.. code:: ipython3
+
+    hexane = mb.load('CCCCCC', smiles=True, name='HEX')
+    hexane.name = 'HEX'
+    hexane.energy_minimize(forcefield='oplsaa' , steps=10**4)
+
+
```

Comment:
It's kind of confusing that the guide is passing in `HEX` to the `name` parameter when creating `hexane`, but then set the name to `HEX` again on the next line. I think we should be consistent for setting `name` for both compounds, maybe just passing it into `mb.load` for both instances?

Reply:
Good Catch here!   I set is as below since it actually failed when I tried it the other way (mb.load('CCCCCC', smiles=True, name = 'HEX')).

hexane = mb.load('CCCCCC', smiles=True)
hexane.name = 'HEX'